examples/AdvancedOperations/AddDynamicSearchAds.cs

```diff
@@ -0,0 +1,320 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V1.Common;
+using Google.Ads.GoogleAds.V1.Errors;
+using Google.Ads.GoogleAds.V1.Resources;
+using Google.Ads.GoogleAds.V1.Services;
+
+using System;
+
+using static Google.Ads.GoogleAds.V1.Enums.AdGroupAdStatusEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.AdGroupStatusEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.AdGroupTypeEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.AdvertisingChannelTypeEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.BudgetDeliveryMethodEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.WebpageConditionOperandEnum.Types;
+using static Google.Ads.GoogleAds.V1.Resources.Campaign.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V1
+{
+    /// <summary>
+    /// This code example adds expanded text ads to a given ad group. To list
+    /// ad groups, run GetAdGroups.cs.
+    /// </summary>
+    public class AddDynamicSearchAds : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddDynamicSearchAds codeExample =
+                new AddDynamicSearchAds();
+            Console.WriteLine(codeExample.Description);
+
+            //The customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            //The ad group ID that contains the ad.
+            long adGroupId = long.Parse("INSERT_ADGROUP_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId, adGroupId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds an expanded text ad that uses advanced features " +
```

Comment:
I think you meant to say 'dynamic search ad' instead of 'expanded text ad' here?

Reply:
Done